scripts/tag_editor_ui/tab_filter_by_selection.py

```diff
@@ -32,7 +32,7 @@ def create_ui(self, image_columns:int):
                 self.btn_add_image_selection = gr.Button(value='Add selection [Enter]', elem_id='dataset_tag_editor_btn_add_image_selection')    
                 self.btn_add_all_displayed_image_selection = gr.Button(value='Add ALL Displayed')    
 
-            self.gl_filter_images = gr.Gallery(label='Filter Images', elem_id="dataset_tag_editor_filter_gallery").style(grid=image_columns)
+            self.gl_filter_images = gr.Gallery(label='Filter Images', elem_id="dataset_tag_editor_filter_gallery", grid=image_columns)
```

Comment:
Incorrect argument to `gr.Gallery`. Should be `columns=image_columns`, not `grid=image_columns`:
```diff
-self.gl_filter_images = gr.Gallery(label='Filter Images', elem_id="dataset_tag_editor_filter_gallery", grid=image_columns)
+self.gl_filter_images = gr.Gallery(label='Filter Images', elem_id="dataset_tag_editor_filter_gallery", columns=image_columns)
```